src/main/java/training/Elbow/ElbowCommandBuilder.java

```diff
@@ -0,0 +1,50 @@
+package training.Elbow;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+
+
+public class ElbowCommandBuilder {
+
+	private final Elbow elbow;
+
+	public ElbowCommandBuilder() {
+		this.elbow = new Elbow();
```

Comment:
Alahamdulila me friendo, not like that it needs to be elbow creating the command builder not opposite

Reply:
I think I got it if you can check this pls